README.md

```diff
@@ -31,6 +31,22 @@ With `reporter: github-pr-review` a comment is added to the Pull Request Convers
 
 Optional. `${{ github.token }}` is used by default.
 
+### `hadolint_flags`
+
+Optional. Pass hadolint flags:
+```
+with:
+  hadolint_flags: --thrusted-repository docker.io
+```
+
+### `hadolint_ignore`
```

Comment:
Please add these flags to the [action.yml](https://github.com/reviewdog/action-hadolint/blob/master/action.yml) file too.

Reply:
Yep just did that, same thoughts.